scipy/interpolate/_bspl.pyx

```diff
@@ -431,21 +436,29 @@ def _make_design_matrix(const double[::1] x,
         B-spline degree.
     extrapolate : bool, optional
         Whether to extrapolate to ouf-of-bounds points.
+    indices : ndarray, shape (n * (k + 1),)
+        Preallocated indices of the final CSR array.
+    indptr : ndarray, shape (n * (k + 1))
+        Preallocated indptr of the final CSR array.
```

Comment:
So indptr just falls through the cython code intact, correct?

Reply:
Yes. Do you prefer to not pass it to the Cython function at all?

Reply:
I think so, yes. A comment would be very welcome, as this becomes a bit unusual. 

An alternative is of course to introduce a pass-through cython function to decide the dtype and dispatch to a fused dtype function. But I think this would be more cumbersome.  So let's do the simplest thing and add a code comment.

Reply:
See https://github.com/scipy/scipy/pull/16840/commits/7a7163f2e40bf684b1c8f4ed242db4039bfb5043.